tests/models/test_model.py

```diff
@@ -647,8 +647,9 @@ def test_mpt_creation(norm_type: str, no_bias: bool, tie_word_embeddings: bool,
 def test_sequence_id_based_masking(attention_impl: str, pos_emb_config: dict):
     # Testing the output of concatenated sequence with sequence id masking vs individual sequences.
     alibi = pos_emb_config['alibi']
-    if alibi and attention_impl == 'flash':
-        pytest.skip(f'alibi only implemented with torch and triton attention.')
+    if alibi and attention_impl == 'flash' and not is_flash_v2_installed(
+            v2_version='v2.4.2'):
+        pytest.skip(f'flash attention below v2.4.2 does not support alibi.')
```

Comment:
I'd make a helper function to handle this logic for all tests.

Reply:
Done.